tests/test_hv.py

```diff
@@ -54,6 +57,41 @@ def test_get_free_ipv4(hv):
     assert str(ipv4_1.ip) == "1.0.0.5"
     assert str(ipv4_2) == "1.0.0.6/24"
 
+
 def test_create_disk_with_options(hv):
-    disk = hv.create_disk('foo_all', size=3, backing_on=True)
+    disk = hv.create_disk("foo_all", size=3, backing_on=True)
     assert isinstance(disk, libvirt.virStorageVol)
+
+
+def test_remove_domain_from_network(hv, domain):
+    NET_XML = """<network>
+    <name>default</name>
+    <uuid>911f54a7-fb02-41ed-9ca1-b35cdc2e6e05</uuid>
```

Comment:
Does possible generate this UUID by code?

Reply:
I'm not sure it makes any sense. The test just ensures the XML is correctly parsed.